packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -391,55 +395,134 @@ export class RafikiService implements IRafikiService {
 
   public async createQuote(receivedQuote: Quote) {
     const feeStructure: Fees = {
-      fixed: 100,
-      percentage: 0.02,
-      asset: 'USD',
-      scale: 2
+      USD: {
+        fixed: 100,
+        percentage: 0.02,
+        scale: 2
+      },
+      EUR: {
+        fixed: 100,
+        percentage: 0.02,
+        scale: 2
+      }
     }
 
+    const isDevelopment = env.NODE_ENV === 'development'
+
+    if (
+      isDevelopment &&
+      receivedQuote.sendAmount.assetCode !==
+        receivedQuote.receiveAmount.assetCode
+    ) {
+      this.deps.logger.info(
```

Comment:
why not use `debug` instead of check if it is development

Reply:
oh lol right